packages/SketchMorph2-Tests.package/MorphicMonetTests.class/instance/testDoesCircleToolShortcutChangeState.st

```diff
@@ -0,0 +1,6 @@
+changing state tests
+testDoesCircleToolShortcutChangeState
+	self subject backend context mode: #pen.
```

Comment:
Law of Demeter?

Reply:
We decided to ignore it for tests since they call methods that are not meant to be called normally.